ci/deps/actions-310.yaml

```diff
@@ -58,6 +60,4 @@ dependencies:
   - zstandard>=0.19.0
 
   - pip:
-    - adbc-driver-postgresql>=0.8.0
-    - adbc-driver-sqlite>=0.8.0
```

Comment:
for my edification, why are these getting moved to a different section? no objection, just curious

Reply:
IIRC making a conda environment and then installing packages via pip is generally not recommended, especially for non pure Python libraries, because pip doesn't necessarily resolve non-Python sub-dependencies correctly all the time.